ibtk/include/ibtk/LaplaceOperator.h

```diff
@@ -67,9 +67,9 @@ class LaplaceOperator : public LinearOperator
     LaplaceOperator(std::string object_name, bool homogeneous_bc = false);
 
     /*!
-     * \brief Destructor.
+     * \brief Default destructor.
```

Comment:
```suggestion
     * \brief destructor.
```
(or `defaulted destructor`)

Reply:
Why not "default"?